basiclti/basiclti-portlet/src/bundle/basiclti.properties

```diff
@@ -3,7 +3,8 @@ noiframes = Your browser does not support iframes.
 noiframe.press.here = Press here to launch.
 not.configured = This tool has not yet been configured.
 
-new.page.launch = Content opened in a new window. Make sure pop up windows are allowed in your browser.
+new.page.launch = (We attempted to launch {0} in a new window. If you wish this to happen automatically in the future you need to configure your browser to allow {1} to open pop-up windows.)
```

Comment:
To launch pop-up windows automatically in the future, configure your browser to allow {1} to open pop-up windows.